roles/coral_credits/defaults/main.yaml

```diff
@@ -0,0 +1,101 @@
+---
+
+# The chart to use
+coral_credits_chart_repo: https://stackhpc.github.io/coral-credits
+coral_credits_chart_name: coral-credits
+coral_credits_chart_version: 0.3.5
+
+# Release information for the coral credits release
+coral_credits_release_namespace: "{{ azimuth_release_namespace | default('azimuth') }}"
+coral_credits_release_name: coral-credits
+
+# The timeout to wait for coral credits API to become ready
+coral_credits_wait_timeout: 10m
+
+# Feature flags
+#   Indicates if coral credits should be enabled
+coral_credits_enabled: yes
```

Comment:
Should default to `no` I think?

Reply:
We can set it to no here, was leaving it enabled to for it to run in CI.

It's also disabled in the config here https://github.com/azimuth-cloud/azimuth-config/pull/158/files

Reply:
Disabled